src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,47 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-of-the-day;
+
+  max-width: $calendar-width;
+  padding: $calendar-pading;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: $height-of-the-day;
+    width: $width-of-the-day;
+
+    background-color: #eee;
+    border: 1px solid black;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+    transition: all 500ms;
```

Comment:
It is recommended to specify the transition properties for better performance. Also, would be great to create a variable for transition-duration